wagtail/contrib/search_promotions/forms.py

```diff
@@ -45,12 +67,31 @@ class Meta:
 
 
 class SearchPromotionsFormSet(SearchPromotionsFormSetBase):
+    """
+    Form set for search promotions
+
+    Attributes
+
+        - minimum_forms (int): Number of forms required for this set.
+    """
+
+    REQUIRED_FIELDS_MISSING = _(
+        "You must recommend a related page OR an external link."
+    )
+    TOO_MANY_FIELDS = _("Please only select a Page OR enter an external link.")
```

Comment:
I only see those two used in a single place in the code. If there is no plan to use the message elsewhere, I think the code would be simpler to understand with the messages inlined?